src/theory/bv/bv_solver.h

```diff
@@ -80,6 +80,9 @@ class BVSolver
     return TrustNode::null();
   }
 
+  /** Additionally collect terms relevant for collecting model values. */
+  virtual void computeRelevantTerms(std::set<Node>& termSet){};
```

Comment:
ClangFormat.

Reply:
This is what clang-format does :)

Reply:
Oh interesting, I just tested it and it seems to be because of the trailing `;`. Without that, it adds the space between `)` and `{}` (which is what I expected).